src/components/pageComponent/login/loginBox.tsx

```diff
@@ -0,0 +1,75 @@
+import ButtonCta from "../../common/buttonCta/buttonCta";
+import Input from "../../common/input/input";
+import style from "./loginBox.module.scss";
+import TopMenuLogin from "./topMenuLogin";
+import {ChangeEvent, FormEvent, useState} from "react";
+import {createLoginToken} from "../../../data/data-global.jsx";
+import Label from "../../common/input/label";
+interface LoginValues {
+  email: string;
+  password: string;
+}
+function LoginBox() {
+  const [loginValues, setLoginValue] = useState<LoginValues>({
+    email: "",
+    password: "",
+  });
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const {name, value} = e.target;
+    setLoginValue((prevLoginValues) => ({
+      ...prevLoginValues, // 이전 복사본을 불러온다
+      [name]: value, // 복사본을 수정한다
+    }));
+  };
+  const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    //const formData = new FormData();
+    //formData.append("email", loginValues.email);
+    //formData.append("password", loginValues.password);
+    createLoginToken(loginValues);
+  };
+  return (
+    <form method="post" className={style.loginBox} onSubmit={handleSubmit}>
```

Comment:
리액트로 폼을 제출할때는 method 속성은 사용하지 않습니다~
```suggestion
    <form className={style.loginBox} onSubmit={handleSubmit}>
```